README.md

```diff
@@ -1 +1,3 @@
 # staywithmeaftercrisis
+
+To pull the docker image for the backend module: `docker pull 3bhady/staywithmeaftercrisis`
```

Comment:
remove this line, you don't need to pull the image, you can re-run docker build and docker run.